hogben/tests/test_fisher.py

```diff
@@ -8,7 +8,7 @@
 import refnx
 import refl1d.experiment
 
-from hogben.simulate import simulate
+from hogben.simulate import SimulateReflectivity
 from hogben.utils import fisher
 from refnx.reflect import SLD as SLD_refnx
```

Comment:
```suggestion
from refnx.reflect import SLD
```
And rename the SLD_refnx to just `SLD` obviously